microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -13,22 +13,28 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
 ## Exhibit G: Terms and Conditions for Microsoft Edge Extensions
 
-These Terms and Conditions for Microsoft Edge Extensions (the "Extensions Addendum") is a supplement to the terms and conditions of the App Developer Agreement, the current version of which can be found at [https://go.microsoft.com/fwlink/p/?LinkID=221922](developer-policies.md).  This Addendum applies if you have submitted or are making available Microsoft Edge Extensions (each, an "Extension") through Microsoft Edge Add-ons.  Except as expressly modified by this Extensions Addendum, all of the terms and conditions in the App Developer Agreement, which is incorporated by reference herein, apply to the offering and distribution of your Microsoft Edge Extensions.  Except where expressly modified by this Addendum, the terms and conditions that apply to an "Application" or "App" under the App Developer Agreement will apply to an Extension as defined in this Addendum and all references to "Store" or "Microsoft Store" under the App Developer Agreement will apply to Microsoft Edge Add-ons as defined in this Addendum.
```

Comment:
@sonakshimore @ashishnirvana1 - 

2x, the link text gives the URL for the Windows app actual "App Developer Agreement", and the body text of present page says "App Developer Agreement" about 10x.  

But, clicking the link takes you instead to the local doc titled "Microsoft Edge Add-ons store developer policies".   Which of the two docs is meant?

Each spot that says "App Developer Agreement" in this page needs to be examined, maybe change all to "Microsoft Edge Add-ons store developer policies".

Reply:
As I posted on another comment:
I believe that the link destination is incorrect. The present article is not an addendum to the store policies, but to the ADA. All evidence is consistent with that except the actual link destination. I'm entering a Suggestion to fix the link presentation & destination.